src/components/Modal/UnhideReview.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { ExclamationCircleOutlined } from "@ant-design/icons";
+import { Col } from "antd";
+import { useTranslation } from "next-i18next";
+import AletheiaButton, { ButtonType } from "../Button";
+import { AletheiaModal, ModalCancelButton } from "./AletheiaModal";
+
+const UnhideReviewModal = ({ visible, isLoading, handleOk, handleCancel }) => {
```

Comment:
Should the unhide action be protected by recaptcha too?

Reply:
in my opinion no, because it is not a form

Reply:
Let's make it protected by recaptcha, since it's a sensitive action.